src/briefcase/console.py

```diff
@@ -41,6 +47,97 @@ class RichConsoleHighlighter(RegexHighlighter):
 
 rich_console = RichConsole(highlighter=RichConsoleHighlighter(), emoji=False)
 
+# Create console that records all logging to save to file later.
+LOG_FILE_WIDTH = 150
+rich_log_file_console = RichConsole(
+    file=open(os.devnull, "w", encoding="utf-8", errors="ignore"),
+    record=True,
+    width=LOG_FILE_WIDTH,
+    no_color=True,
+    markup=False,
+    emoji=False,
+    highlight=False,
+    soft_wrap=True,
+)
+
+
+def _print_proxy(message="", *, show=True, stack_offset=1, **kwargs):
+    """Funnel for all printing to ensure all logging is recorded for
+    potentially saving to a file later regardless if it is printed to the
+    screen or not.
+
+    :param message: content to print/log
+    :param show: True (default) to print message to the screen; False to only record message
+    :param stack_offset: number of stack levels to where logging was invoked
+    """
+    if show:
+        rich_console.print(message, **kwargs)
+    rich_log_file_console.log(message, _stack_offset=stack_offset, **kwargs)
+
+
+def cleanup_log_files(command):
+    """Delete old log files."""
+    maximum_log_file_age = 14  # days
+    age_limit = datetime.now().timestamp() - (maximum_log_file_age * 24 * 60 * 60)
+    log_files = next(command.os.walk(command.logs_path), (None, None, []))[2]
+    for log_file in (command.logs_path / file for file in log_files):
+        if log_file.stat().st_mtime < age_limit:
+            try:
+                log_file.unlink()
+            except OSError:
+                pass
```

Comment:
Wasn't sure if it was worth logging that a previously created log file couldn't be deleted....it seems like a strange situation...alternatively, we could nag the user to delete them themselves...